packages/wonder-blocks-form/src/components/__tests__/field-heading.test.tsx

```diff
@@ -187,7 +187,11 @@ describe("FieldHeading", () => {
         render(
             <FieldHeading
                 field={<TextField id="tf-1" value="" onChange={() => {}} />}
-                label={<I18nInlineMarkup>Hello, world!</I18nInlineMarkup>}
+                label={
+                    <I18nInlineMarkup b={(s: string) => <b>{s}</b>}>
```

Comment:
Caught this and the next file during testing - they weren't using I18nInlineMarkup correctly, but that fixes them so that they are now rendered with markup.